source/edge-server/configure.txt

```diff
@@ -134,271 +115,162 @@ EC2, you must open ports to accept incoming connections:
 - Development: Use HTTP on port 80 for simplicity and speed of development.
 - Production: Use SSL with port 443 to secure access to production data.
 
-Edge Server supports TLS. You can find configuration details in the 
-"Complete the Edge Server Configuration Details" section on this page.
+Edge Server supports TLS.
 
-.. _edge-server-os-support:
-
-OS Compatibility
-~~~~~~~~~~~~~~~~
-
-The Edge Server has been verified with the following operating systems:
-
-- Ubuntu Server 18+
-- macOS 11.x and newer running on M1 or Intel 2018+ MacBook Pros
-
-Other operating systems may be possible, but have not yet been verified.
-
-.. _edge-server-hardware-requirements:
-
-Hardware Requirements
-~~~~~~~~~~~~~~~~~~~~~
-
-Edge Server benchmarking is not yet complete, so minimum required hardware
-is not yet available. 
+[PLACEHOLDER: Add a link to details about configuring TLS once engineering
+decides how/where to do that.]
 
 .. _edge-server-install-and-configure:
 
-Install and Configure the Edge Server
--------------------------------------
+Install and Configure the Edge Server Instance
+----------------------------------------------
 
-Set up and run the Edge Server on the host.
+After creating an Edge Server Service or Device Sync Service, you must create 
+an Edge Server instance. Each service may have many Edge Server instances. When 
+you create an Edge Server instance, you get an installation script you can use 
+to download the Edge Server code, and a token you can use to register the 
+Edge Server instance. For more details, refer to 
+:ref:`create-edge-server-instance`.
+
+Use the installation script to set up and run the Edge Server instance on the 
+host.
 
 .. include:: /includes/important-stop-tiered-sync-server.rst
 
 .. procedure::
 
-   .. step:: Get the Edge Server Code
+   .. step:: Install the Edge Server Code
+
+      Edge Server uses a command-line tool called ``edgectl`` to manage
+      the Edge Server instance on the host hardware. This includes an install
+      script to manage the installation and install required dependencies.
 
-      Use ``wget`` to get the current Edge Server code as a ``.tar`` file:
+      Use this command to install the command-line tool and the Edge Server 
+      code:
 
       .. code-block:: shell
 
-         wget --content-disposition https://services.cloud.mongodb.com/api/client/v2.0/tiered-sync/package/latest
+         curl -s http://localhost:8080/edge/install.sh | bash -s -- --app-id="<INSERT-YOUR-APP-ID-HERE>" --platform=compose
```

Comment:
this should be services.cloud.mongodb.com

Reply:
if they are copy-pasting from here, you dont need the extra params. you can just do 
```
curl -s http://localhost:8080/edge/install.sh | bash
```

Reply:
the script will then prompt the user to fill them in

Reply:
Alright, updating - thanks!